SPECS/usbip/usbip.spec

```diff
@@ -2,15 +2,15 @@
 
 Name:		   usbip
 Summary:	   USB/IP user-space
-Version:	   5.15.34.1
-Release:	   3%{?dist}
+Version:	   6.6.14.1
+Release:	   1%{?dist}
 License:	   GPLv2+
 Vendor:            Microsoft Corporation
 Distribution:   Azure Linux
 Group:             System/Kernel
-# https://github.com/microsoft/CBL-Mariner-Linux-Kernel/archive/rolling-lts/mariner-2/%%{version}.tar.gz
+# https://github.com/microsoft/CBL-Mariner-Linux-Kernel/archive/rolling-lts/mariner-%%{azl}/%%{version}.tar.gz
 # In the interests of keeping the source rpm from being ridiculously large,
-# download the Linux kernel from above and run `extract_usbip.sh <version>`
+# download the Linux kernel from above and run `extract_usbip.sh <version> <mariner_version>`
```

Comment:
```suggestion
# download the Linux kernel from above and run `extract_usbip.sh <version> <azure_linux_version>`
```

:)

Reply:
Ah, good point. This was copied from `kernel.spec` which still uses `%define mariner_version 3`. Forgot to update when I moved to `%{azl}`